src/repl.rs

```diff
@@ -1,7 +1,13 @@
 use std::error::Error as StdError;
+use std::fmt;
+use std::sync::Mutex;
+
+use rustyline::error::ReadlineError;
```

Comment:
Maybe scope the crate import to this module, seeing as it is not used elsewhere? Ie.

```rust
extern crate rustyline;
use self::rustyline::error::ReadlineError;

...

let editor = rustyline::Editor::new();
```